### ISSUE TITLE
Some paper nodes' representations in CORA are all zero vectors

### ISSUE THREAD

**Issue by Wentao-Xu** [huazai1992/HGCN#2]
Thanks for sharing the code!
When I observe the node representations in '/data/net_cora/P.feat.npz', I find 1347 paper nodes' representations are all zero vectors (there are a total of 19396 paper nodes in CORA).

**Comment by huazai1992** [comment]
We extract a bag-of-words representation of all the paper titles as content feature for each paper node, which includes 300 words (not all). So, there exist some paper nodes that are not contain these words.

**Comment by huazai1992** [comment]
We extract a bag-of-words representation of all the paper titles as content feature for each paper node, which includes 300 words (not all). So, there exist some paper nodes that do not contain these words.

**Comment by Wentao-Xu** [comment]
Thanks for your reply. 
Would this affect the performance? Since 6.94% of nodes do not have features. And how do you split the training and test set? How many nodes in the test set?

**Comment by huazai1992** [comment]
I think it would not affect the performance. In our paper, we mainly use one-hot codes of the labels of target-type nodes as the features (i.e., *.feat.label). For all methods, we randomly sample 80% of labeled nodes as the training set and 20% as the test set

**Comment by Wentao-Xu** [comment]
OK, thanks for your reply!
